src/app/pages/non-pass-finalisation/non-pass-finalisation.analytics.effects.ts

```diff
@@ -193,4 +196,27 @@ export class NonPassFinalisationAnalyticsEffects {
     }),
   ));
 
+  NonPassFinalisationReportActivityCode$ = createEffect(() => this.actions$.pipe(
```

Comment:
can we make this lower camel case to be consistent with other effects please i.e.

```nonPassFinalisationReportActivityCode$```